packages/vanilla/src/sass/components/footer/_internal.scss

```diff
@@ -161,11 +155,6 @@
     }
   }
 
-  .cbp-footer__info {
-    padding-top: var(--cbp-space-4x);
-    padding-bottom: var(--cbp-space-4x);
-  }
```

Comment:
This spacing probably from before the mock ups changed, looking at mock ups now it looks like there needs to be a padding top for the `.cbp-footer__info` class at `var(--cbp-space-4x)` across all screen widths. The bottom padding however should be verified with Jeremy (@yellowzapp) since that was spaced out for the action bar.

Reply:
Jeremy's specific comment said that it should be changed to 5x. We can address the action bar issue when we have it and determine its functionality. I definitely wouldn't put a bunch of extra space at the bottom without the floating action bar there.

Reply:
Oh if thats the case then it should what you changed it to.